scale/recipe/models.py

```diff
@@ -737,45 +388,6 @@ def get_details(self, recipe_id):
         recipe.sub_recipe_types = RecipeType.objects.all().filter(id__in=sub_ids)
         return recipe
 
-    # TODO: remove function when REST API v5 is removed
-    def get_details_v5(self, recipe_id):
-        """Gets the details for a given recipe including its associated jobs and input files.
-
-        :param recipe_id: The unique identifier of the recipe to fetch.
-        :type recipe_id: :int
-        :returns: A recipe with additional information.
-        :rtype: :class:`recipe.models.Recipe`
-        """
-
-        # Attempt to fetch the requested recipe
-        recipe = Recipe.objects.select_related(
-            'recipe_type', 'recipe_type_rev', 'event', 'event__rule', 'root_superseded_recipe',
-            'root_superseded_recipe__recipe_type', 'superseded_recipe', 'superseded_recipe__recipe_type',
-            'superseded_by_recipe', 'superseded_by_recipe__recipe_type'
-        ).get(pk=recipe_id)
-
-        # Update the recipe with source file models
-        input_file_ids = []
-        for file_value in recipe.get_input_data().values.values():
-            if file_value.param_type != FileParameter.PARAM_TYPE:
-                continue
-            input_file_ids.extend(file_value.file_ids)
-        input_files = ScaleFile.objects.filter(id__in=input_file_ids)
-        input_files = input_files.select_related('workspace').defer('workspace__json_config')
-        input_files = input_files.order_by('id').distinct('id')
-
-        interface = recipe.recipe_type_rev.get_definition().input_interface
-        recipe_def_dict = convert_recipe_definition_to_v1_json(recipe.recipe_type_rev.get_definition()).get_dict()
-        recipe_data_dict = convert_data_to_v1_json(recipe.get_input_data(), interface).get_dict()
-        recipe.inputs = self._merge_recipe_data(recipe_def_dict['input_data'], recipe_data_dict['input_data'],
-                                                input_files)
-
-        # Update the recipe with job models
-        jobs = RecipeNode.objects.filter(recipe_id=recipe.id)
-        jobs = jobs.select_related('job', 'job__job_type', 'job__event', 'job__error')
-        recipe.jobs = jobs
-        return recipe
-
     def process_recipe_input(self, recipe):
         """Processes the input data for the given recipe to populate its input file models and input meta-data fields.
         The caller must have obtained a model lock on the given recipe model.
```

Comment:
Same as above, also I see a legacy trigger TODO

Reply:
This legacy trigger looks more involved than just removing a comment. I'll create an issue to address this after the PR is merged.